src/Microsoft.ML.Data/Transforms/Normalizer.cs

```diff
@@ -102,35 +102,35 @@ internal static ColumnOptionsBase Create(string outputColumnName, string inputCo
             }
         }
 
-        public abstract class FixZeroColumnOptionsBase : ColumnOptionsBase
+        public abstract class ControlZeroColumnOptionsBase : ColumnOptionsBase
```

Comment:
>ControlZeroColumnOptionsBase [](start = 30, length = 28)

why change this?

Reply:
`FixZero` is a bool option applied to zeros. Those zeros may not be fixed if `FixZero=false`. Thus, I feel `ControlZero` sounds more appropriate.

---
In reply to: [264105076](https://github.com/dotnet/machinelearning/pull/2888#discussion_r264105076) [](ancestors = 264105076)

Reply:
I actually think that `FixZero` makes more sense because it's "fixing" something in place (and not "fixing" a problem). It is a bit more precise than "control".

---
In reply to: [264343033](https://github.com/dotnet/machinelearning/pull/2888#discussion_r264343033) [](ancestors = 264343033,264105076)

Reply:
Ok. Let me rename it back.

---
In reply to: [264825031](https://github.com/dotnet/machinelearning/pull/2888#discussion_r264825031) [](ancestors = 264825031,264343033,264105076)

Reply:
`FixZero=true` means that I fixe the value of zero in place. `FixZero=false` means don't fix zero in place. That seems consistent, doesn't it?

---
In reply to: [264868717](https://github.com/dotnet/machinelearning/pull/2888#discussion_r264868717) [](ancestors = 264868717,264825031,264343033,264105076)

Reply:
The problem is not only for that flag's name. When appearing in the name of `FixZeroColumnOptionsBase`, no one knows `FixZero` is a Boolean variable. Your judgement is valid only when `FixZero` is a field.

---
In reply to: [265342165](https://github.com/dotnet/machinelearning/pull/2888#discussion_r265342165) [](ancestors = 265342165,264868717,264825031,264343033,264105076)